js/libs/keycloak-admin-client/src/defs/effectiveMessageBundleRepresentation.ts

```diff
@@ -0,0 +1,8 @@
+export default interface EffectiveMessageBundleRepresentation {
+  theme?: string;
+  themeType?: string;
+  locale?: string;
+  hasWords?: string;
+  source?: boolean;
+  error?: string;
```

Comment:
When is this `error` data returned? If the API gives a [non-ok](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) code this type will be ignored. So if this is part of the error response only, it can be omitted.

Reply:
Thanks Jon, we can omit it.